common/datavalues/src/types/data_type.rs

```diff
@@ -75,6 +78,7 @@ impl DataType {
                 ArrowDataType::Struct(arrows_fields)
             }
             String => ArrowDataType::LargeBinary,
+            Interval(_) => ArrowDataType::Float64,
```

Comment:
Why interval is Float64?

Reply:
Not necessary, it can be float32 because I hope it can support something like "interval '1.5' year".
And I don;t think this line will be executed.

Reply:
But I think UInt64 will cover if we store intervals as NanoSeconds.

Reply:
ClickHouse seems to store int64, because negative interval needed.

Reply:
Sure. No problem. how about Int64 store seconds number ?

Reply:
Why not use arrow Interval type?

Reply:
Seconds is ok,   "interval '1.5' year" we can cast `1.5 * Interval` --> `Float64` --> `Int64 interval`

Reply:
> Why not use arrow Interval type?

The idea is from clickhouse, they don't have Interval as a built-in column type, but more like a syntax sugar. So the interval data won't be stored into parquet. -- This is my naive understanding, correct me if I am wrong.

Reply:
😂 In ClickHouse, the interval may be complex:

There may be two ClickHouse interval types(DataTypeNumber and DataTypeInterval). For their functions processing, in https://github.com/ClickHouse/ClickHouse/blob/f559c3411392901da6d5130711e1637fb22d2cbd/src/Functions/FunctionBinaryArithmetic.h#L599, but we may be better and more elegant than ClickHouse (I'm not sure)

Reply:
> Why not use arrow Interval type?

question: will datefuse support Interval type as sql data type, Like "CREATE TABLE t (c1 interval);"   If yes, I think we should use the arrow interval type for maybe storage. Notice that arrow also support Date32 and Date64, but we we using uint32 for Date.